account_analytic_parent/views/account_analytic_account_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--
+        License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+    -->
+    <record id="view_account_analytic_account_form" model="ir.ui.view">
+        <field name="name">account.analytic.account.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form" />
+        <field name="arch" type="xml">
+            <field name="plan_id" position="after">
+                <field name="parent_id" string="Parent Analytic Account" />
+            </field>
```

Comment:
```suggestion
            <group name="main" position="attributes">
                <attribute name="string">General</attribute>
            </group>
            <group name="main" position="before">
                <group name="parent" col="2">
                    <field name="parent_id" class="oe_inline"/>
                </group>
            </group>
```
As a general improvement. I guess that it makes more sense to have the view in a similar structure as in the Product Category, where we also have a Parent-Child relation.
In order to adapt, we also could name the original group with the "General" string. What do you think?